egs2/mini_an4/enh1/conf/train_with_dynamic_mixing.yaml

```diff
@@ -0,0 +1,32 @@
+
+encoder: stft
+encoder_conf:
+    n_fft: 512
+    hop_length: 128
+decoder: stft
+decoder_conf:
+    n_fft: 512
+    hop_length: 128
+separator: rnn
+separator_conf:
+    rnn_type: blstm
+    num_spk: 2
+    nonlinear: relu
+    layer: 1
+    unit: 128
+    dropout: 0.2
+
```

Comment:
Can you add a comment about this configuration?

Reply:
done.